app/src/main/kotlin/com/simplemobiletools/camera/activities/MainActivity.kt

```diff
@@ -544,6 +564,27 @@ class MainActivity : SimpleActivity(), PhotoProcessor.MediaSavedListener, Camera
 
     override fun onMediaCaptured(uri: Uri) {
         loadLastTakenMedia(uri)
+        ensureBackgroundThread {
+            if (isImageCaptureIntent()) {
+                val bitmap = contentResolver.loadThumbnail(uri, Size(30, 30), null)
```

Comment:
Would need a better way to generate the bitmap

Reply:
why dont you use Glide for it?